cmd/neofs-cli/internal/commonflags/flags.go

```diff
@@ -82,3 +83,26 @@ func Bind(cmd *cobra.Command) {
 	_ = viper.BindPFlag(RPC, ff.Lookup(RPC))
 	_ = viper.BindPFlag(Timeout, ff.Lookup(Timeout))
 }
+
+// AcceptTimeout makes given cobra.Command to accept 'timeout' flag as global
+// execution timeout. If cobra.Command.Run is specified, AcceptTimeout precedes
+// it with time limited context setting.
+func AcceptTimeout(cmd *cobra.Command) {
```

Comment:
Why not `PreRun` for it?

Reply:
cuz then context cancellation must be done in `PostRun`, and afaik cobra doesn't provide mechanism to sync stuff b/w pre-run/post-run funcs. Well, we can use global variables, and i guess they'll fit, but i tried to avoid them